docs/Local_Login.md

```diff
@@ -0,0 +1,15 @@
+# How To Get Login Working Locally
+
+## Step-by-step Guide
+
+1. Run the shell script, `generate_ssl_certs.sh` located in the login folder.
+
+2. You will also need to create a `idp_key.pem` file in the certs folder that gets created with the following key:
+
+`MIIDXTCCAkWgAwIBAgIJALmVVuDWu4NYMA0GCSqGSIb3DQEBCwUAMEUxCzAJBgNVBAYTAkFVMRMwEQYDVQQIDApTb21lLVN0YXRlMSEwHwYDVQQKDBhJbnRlcm5ldCBXaWRnaXRzIFB0eSBMdGQwHhcNMTYxMjMxMTQzNDQ3WhcNNDgwNjI1MTQzNDQ3WjBFMQswCQYDVQQGEwJBVTETMBEGA1UECAwKU29tZS1TdGF0ZTEhMB8GA1UECgwYSW50ZXJuZXQgV2lkZ2l0cyBQdHkgTHRkMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAzUCFozgNb1h1M0jzNRSCjhOBnR`
```

Comment:
If it's always this key, should we include it in our repo?

Reply:
I don't necessarily think so, there is probably a way to create a certification with this file included, which is probably a better option with the generate script section.  However, having a certificate actually lingering may be a potential security risk to someone who may not be aware that it is there?

Reply:
I made the other edits to fix the documentation, I'm unsure about this one as, like James mentioned, it could be a security risk.

Reply:
Let's do this for now, and we won't use it in production.  For dev, it's fine.  We'll come back to it later when we deploy and figure it out.